src/app/components/editors/note-editor/terra-note-editor.component.ts

```diff
@@ -102,4 +102,10 @@ export class TerraNoteEditorComponent implements OnInit, ControlValueAccessor
     {
         this._onTouchedCallback = fn;
     }
+
+    public focus():void
+    {
+        document.getElementsByClassName('ql-editor')[0].id = 'editor-focused';
```

Comment:
Das geht so leider nicht. Es können mehr als nur ein Note-Editor geschweige denn ein Quill-Editor im DOM sein. Damit kriegst du also nicht unbedingt den Editor, den du haben will. Nutze auch hier lieber den ViewChild-Decorator um auch nach dem Editor in dem entsprechenden Scope zu suchen.